src/olympia/reviewers/views.py

```diff
@@ -136,6 +137,101 @@ def beta_signed_log(request):
+        )]
+    if acl.action_allowed(request, amo.permissions.ADDONS_POST_REVIEW):
+        sections[ugettext('Auto-Approved Add-ons')] = [(
+            ugettext('Auto Approved Add-ons ({0})').format(
+                AutoApprovalSummary.get_auto_approved_queue().count()),
+            reverse('reviewers.queue_auto_approved')
+        ), (
+            ugettext('Performance'),
+            reverse('reviewers.performance')
+        ), (
+            ugettext('Add-on Review Log'),
+            reverse('reviewers.reviewlog')
+        )]
+    if acl.action_allowed(request, amo.permissions.ADDONS_CONTENT_REVIEW):
+        sections[ugettext('Content Review')] = [(
+            ugettext('Content Review ({0})').format(
+                AutoApprovalSummary.get_content_review_queue().count()),
+            reverse('reviewers.queue_content_review')
+        ), (
+            ugettext('Performance'),
+            reverse('reviewers.performance')
+        )]
+    if acl.action_allowed(request, amo.permissions.THEMES_REVIEW):
+        sections[ugettext('Lightweight Themes')] = [(
+            ugettext('New Themes ({0})').format(
+                Persona.objects.filter(
+                    addon__status=amo.STATUS_PENDING).count()),
+            reverse('reviewers.themes.list')
+        ), (
+            ugettext('Themes Updates ({0})').format(
+                RereviewQueueTheme.objects.count()),
+            reverse('reviewers.themes.list_rereview')
+        ), (
+            ugettext('Flagged Themes ({0})').format(
+                Persona.objects.filter(
+                    addon__status=amo.STATUS_REVIEW_PENDING).count()),
+            reverse('reviewers.themes.list_flagged')
+        ), (
+            ugettext('Themes Review Log'),
+            reverse('reviewers.themes.logs')
+        ), (
+            ugettext('Deleted themes Log'),
+            reverse('reviewers.themes.deleted')
+        )]
+    if acl.action_allowed(request, amo.permissions.RATINGS_MODERATE):
+        sections[ugettext('User Ratings Moderation')] = [(
+            ugettext('Ratings Awaiting Moderation ({0})').format(
+                Rating.objects.all().to_moderate().count()),
+            reverse('reviewers.queue_moderated')
+        ), (
+            ugettext('Moderated Review Log'),
+            reverse('reviewers.eventlog')
+        )]
+    if acl.action_allowed(request, amo.permissions.ADDONS_REVIEW_UNLISTED):
+        sections[ugettext('Unlisted Add-ons')] = [(
+            ugettext('All Unlisted Add-ons'),
+            reverse('reviewers.unlisted_queue_all')
+        )]
+
+    if acl.action_allowed(request, amo.permissions.ADDON_REVIEWER_MOTD_EDIT):
+        sections[ugettext('Announcement')] = [(
+            ugettext('Update message of the day'),
+            reverse('reviewers.motd')
+        )]
+    return render(request, 'reviewers/dashboard.html', base_context(**{
+        # base_context includes motd.
+        'sections': sections
+    }))
```

Comment:
Does `REVIEWER_TOOLS_VIEW` not get anything?

Reply:
No :) But the users that have this permission might have more perms that allow them to see relevant sections

Reply:
hmm... But if they have other permissions then why do they need REVIEWER_TOOLS_VIEW?  People who get it specifically _don't_ have write permissions such as ADDONS_REVIEW, right?

Reply:
mmm I don't know. I assumed that they'd have more - it wasn't specified in the issue what do for for them.

Reply:
I wasn't aware of this permission until recently, when @wagnerand brought it up. I think it could be useful to keep it as a permission that gives you read-only access to *all* reviewer tools, in which case someone with that permission would see the entire dashboard.